assets/components/flashSaleCountdown/flashSaleCountdown.scss

```diff
@@ -0,0 +1,20 @@
+.component-flash-sale-countdown__chip {
+  font-family: $gu-text-sans-web;
+  font-size: 18px;
+  padding: 6px 10px 4px;
+  display: inline-block;
+}
+
+.component-flash-sale-countdown__chip--time {
+  background: gu-colour(sport-garnett-feature);
+  color: #fff;
+  font-weight: 800;
+  width: 98px;
```

Comment:
This fixed width is set to prevent the counter from shifting sizes slightly when the time changes, as the font doesn't have monospaced numbers.